client/src/components/profile/UserInfoCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Card, Image } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar, faStarHalf } from '@fortawesome/free-solid-svg-icons';
+import { faFacebook, faInstagram, faWhatsapp } from '@fortawesome/free-brands-svg-icons';
+
+import profileImg from '../../assets/profileImg.png';
+import 'antd/dist/antd.css';
+import '../../App.css';
+
+function UserInfoCard() {
```

Comment:
build your component to receive props,
and In the place you call add these data as props